tests/bdd/steps/given/openUrl.js

```diff
@@ -2,7 +2,7 @@ const { Given } = require("cucumber")
 const { getBrowserContext } = require('../../support/setup')
 const { getPage } = getBrowserContext()
 
-Given("I open the site {site}", async site => {
+Given('I open the site {word}', async site => {
```

Comment:
When I tried to use QAWolf to edit this file, I got all kinds of errors => http://snpy.in/CrYVGX
* Seems like it's not stopping Jest execution
* Not really sure why

I tried it with a few different variations, and always get the same erros
* No browser running
* Browser running
* Running the test command with out qawolf ( This works )
  * Then running the command with QAWolf, and it breaks again

It looks to be running QAWolf, but not stopping
* So the feature steps get treated as  user actions by QAWolf, and they are recorded
* See here => http://snpy.in/QiEcEH

Reply:
Fixed.

The issue here was the timeout was stuck at 5000ms. I made it a large number instead (jest doesn't seem to support a way to specifically disable timeouts, and the `Infinity` value didn't work either)